src/main/scala/com.snowplowanalytics/iglu.schemaddl/jsonschema/Linter.scala

```diff
@@ -351,6 +351,27 @@ object Linter {
       }
   }
 
+  final case object schemaUri extends Linter {
+    self =>
+
+    val expectedSchemaUri = "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#"
```

Comment:
Could you please define it somewhere in `Schema` object?

```scala
val SelfDescribingUri = "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#"
```

As I already mentioned, this is a very important constant, I'm surprised we don't have it in core.